detection_rules/config.py

```diff
@@ -149,6 +188,14 @@ class RulesConfig:
     action_dir: Optional[Path] = None
     exception_dir: Optional[Path] = None
 
+    def __post_init__(self):
+        """Perform post validation on packages.yml file."""
+        if 'package' not in self.packages:
+            raise SystemExit('Missing the `package` field defined in packages.yml.')
+
+        if 'name' not in self.packages['package']:
+            raise SystemExit('Missing the `name` field defined in packages.yml.')
```

Comment:
I would raise `ValueError` here and then below, try/except and catch it and _then_ raise `SystemExit` from there

```suggestion
        if 'package' not in self.packages and  'name' not in self.packages['package']:
            raise ValueError('Missing required `package.name` value within: {self.packages_file}.')
```

Reply:
makes sense!